hack/jenkins/e2e-runner.sh

```diff
@@ -257,12 +257,19 @@ case "${KUBERNETES_PROVIDER}" in
             cp /var/lib/jenkins/gce_keys/google_compute_engine ${WORKSPACE}/.ssh/
             cp /var/lib/jenkins/gce_keys/google_compute_engine.pub ${WORKSPACE}/.ssh/
         fi
-        if [[ ! -f ${WORKSPACE}/.ssh/google_compute_engine ]]; then
-            echo "google_compute_engine ssh key missing!"
+        echo 'Checking existence of private ssh key'
+        local _key="${WORKSPACE}/.ssh/google_compute_engine"
+        if [[ ! -f "${_key}" || ! -f "${_key}.pub" ]]; then
+            echo 'google_compute_engine ssh key missing!'
             exit 1
         fi
+        echo "Checking presence of public key in ${PROJECT}"
+        if ! gcloud compute "--project=${PROJECT}" project-info describe |
```

Comment:
I would agree were I trying to parse the output, but I'm just trying to ensure the public key exists.

I would rather avoid trying to process the json output and validate that
```
output = json.loads(gcloud_output)
for key in output['commonInstanceMetadata']['items']['sshKeys'].split('\n'):
  if public_key in key: return True
else:
  return False
```

Seems overkill here (and annoying to do in bash)